src/popup/components/annotationList/AnnotationList.tsx

```diff
@@ -49,15 +54,22 @@ export default class AnnotationList extends React.Component<Partial<IAnnotationL
     } else {
       return (
         <div className={styles.self}>
-          <button onClick={this.handleGoBackClick}>Wróć</button>
-          <ul>
+          <div onClick={this.handleGoBackClick}>
+            <Icon className={styles.chevronButton} icon={ic_chevron_left} size={25}/>
+          </div>
+          <ul className={styles.annotationList}>
```

Comment:
It would be nice to ensure constant width for the case when the list is being loaded (a few lines above) and also include some basic styling for loading.